src/main/java/io/relution/jenkins/awssqs/SQSTriggerQueue.java

```diff
@@ -530,7 +530,7 @@ public FormValidation doValidate(
 
                 final String url = result.getQueueUrl();
                 if ((credentialsId == null) || credentialsId.isEmpty())    {
-                    return FormValidation.error("No credentials set");
+                    return FormValidation.ok("Access to %s successful\n(%s)", queue.getName(), url);
```

Comment:
This is unrelated to the CasC changes, but if `getQueueUrl()` succeeded then the lack of credentials is irrelevant (i.e. we're probably using an EC2 instance profile) and not an actual failure.